src/window/update_loop.rs

```diff
@@ -20,25 +21,44 @@ enum FocusedState {
     Unfocused,
 }
 
+const MAX_ANIMATION_DT: f32 = 1.0 / 120.0;
```

Comment:
How is/should this constant be chosen? If I understand correctly, it effectively controls the number of simulation steps further below, but is there a specific thought behind setting this to be fixed at `1 / 120`? Should this be configurable at runtime?

Reply:
It's just the maximum animation frame length, it needs to be small enough so that the euler integration used for the animation doesn't blow up due to numerical errors. And big enough so that we don't burn too much CPU time. 120 seems to work well experimentally, but I think 100 or 200 would work just as well.

The frame dt is then divided into equally sized steps, so that:
1. If the FPS >= 120 you get one animation frame per real frame
2. 60 <= FPS < 120  , you get two animation frames per real frame
3. 40 <= FPS < 60, you get three animation frame per real frame

and so on.

Now that I think about it, 130 would probably make more sense, since very few monitors run at that frequency, so there's less chance of jumping between the 1 or 2 animation frames per frame.

Reply:
Can I modify this max FPS? I noticed that the latest commit locks it at 138 on my device, whereas before it could exceed 500. Just a question, it already looks good enough now!

Reply:
@fecet,
There was a bug before, it did not respect `--vsync` (the default) on X11, and macOS.

With that it locks the refresh rate to the same as the monitor, which almost always looks the best, and uses less power and computer resources.

But if you really want, you can manually start Neovide with `neovide --novsync`. And then inside Neovide `:let g:neovide_refresh_rate=500` to set it to 500 for example.

BTW, a good way of profiling how fast your computer is able to render, is to start `neovide --novsync --noidle`, which forces neovide to render, even when there are no changes. Then set `:let g:neovide_refresh_rate=9999`, so something big that is not achievable. You can then see the frame rate with `:let g:neovide_profiler=1`.

Reply:
hank you for your detailed explanation, this method works very well!

Reply:
Interesting, so I have a MBP m2 which is 120 refresh rate. I had this set:

```
  vim.g.neovide_refresh_rate = 120
```

and tried that profiler and it looked to be about 110 but jumping around a lot

then i put 

```
  vim.g.neovide_refresh_rate = 130
```

per what you just said and now its a very stable 120, where i can at least read it (before it was changing +/- every ms)

Thanks for that tip

Reply:
Another interesting thing is vsync default goes to 60 while monitor is able to do 120. so i suppose in my case i do require not being in vsync mode and using the 130 refresh rate setting. 🤷🏻‍♂️

Reply:
@9mm. It looks like the bug @MultisampledNight did not properly fix it on macOS then. So, in that case we still need to implement https://developer.apple.com/documentation/corevideo/cvdisplaylinkoutputcallback to get it properly working on macOS.

Reply:
And 130 was meant for the `MAX_ANIMATION_DT` constant, so a completely different thing, and not meant for `neovide_refresh_rate`, but I'm happy it worked anyway.

Reply:
No! Sorry, I'm not on this branch. I was commenting in general, sorry, I should have said that... it is 5am

Reply:
I was definitely considering switching to main branch though to help try some of this new stuff with osx perspective